docs/modules/ROOT/pages/installation/installation.adoc

```diff
@@ -10,6 +10,7 @@ To use it, you need to be connected to a cloud environment or to a local cluster
 Camel K installation is usually straightforward, but for certain cluster types you need to apply specific configuration settings
 before installing it. Customized instructions are needed for the following cluster types:
 
+- xref:installation/crc.adoc[Red Hat CodeReady Containers (CRC)]
```

Comment:
Can you move this to after minishift. Minikube should be first.

Reply:
Thanks @davsclaus for reviewing! Done